h2o-docs/src/booklets/v2_2015/source/GLMBooklet.tex

```diff
@@ -571,6 +571,120 @@ \subsubsection{Tweedie Models}
 \centering
 \includegraphics[scale=0.5]{../../../product/images/tweedie_deviance.png}
 \end{figure}
+
+
+\paragraph{Tweedie Dispersion Parameter Estimation}
+
+The density for the maximum likelihood function for Tweedie can be written as:
+
+\begin{equation} \label{eq:1}
+f( y; \theta, \phi) = a (y, \phi, p) \exp \Big[ \frac{1}{\phi} \big\{ y \theta - k(\theta) \big\} \Big]
+\end{equation}
+
+where:
+
+\begin{itemize}
+\item $a (y, \phi, p), k(\theta)$ are suitable known functions
+\item $\phi$ is the dispersion parameter and is positive
+\item $\theta = \begin{cases} \frac{\mu ^{1-p}}{1-p} & p \neq 1 \\ \log (\mu) & p = 1 \\\end{cases}$
+\item $k(\theta) = \begin{cases} \frac{\mu ^{2-p}}{2-p} & p \neq 2 \\ \log (\mu) & p=2 \\\end{cases}$
+\item the value of $\alpha (y,\phi)$ depends on the value of $p$
+\end{itemize}
+
+If there are weights introduced to each data row, equation \ref{eq:1} will become:
+
+$$f \Big( y; \theta, \frac{\phi}{w} \Big) = a \Big( y, \frac{\phi}{w}, p \Big) \exp \Big[ \frac{w}{\phi} \big\{ y\theta - k(\theta) \big\} \Big]$$
+
+\subparagraph{$\alpha (y, \phi) when 1<p<2$}
+
+For $Y=0$,
+
+\begin{equation} \label{eq:2}
+P(Y=0) = \exp \Big\{-\frac{\mu^{2-p}}{\phi (2-p)} \Big\}
+\end{equation}
+
+For $Y>0$,
+
+\begin{equation} \label{eq:3}
+a(y, \phi, p) = \frac{1}{y} W(y, \phi, p)
+\end{equation}
+
+with $W(y, \phi, p) = \sum^{\infty}_{j=1} W_j$ and 
+
+\begin{equation} \label{eq:4}
+W_j = \frac{y^{-j \alpha}(p-1)^{\alpha j}}{\phi^{j(1-\alpha)} (2-p)^j j!T(-j\alpha)}
+\end{equation}
+
+If weight is applied to each row, equation \ref{eq:4} becomes:
+
+\begin{equation} \label{eq:5}
+W_j = \frac{w^{j(1-\alpha)}y^{-j \alpha}(p-1)^{\alpha j}}{\phi^{j(1-\alpha)}(2-p)^j j!T(-j \alpha)}
+\end{equation}
+
+The $W_j$ terms are all positive. The following figure plots for $\mu = 0.5, p=1.5, \phi =1. y=0.1$.
+
+\begin{figure}[h]
+\centering
+\includegraphics[scale=0.25]{images/dispersion_param_fig1.png}
+\end{figure}
+
+\subparagraph{$\alpha (y, \phi) \text{ when } p > 2$}
+
+Here, you have,
+
+\begin{equation} \label{eq:6}
+a(y, \phi, p) = \frac{1}{\pi y}V(y,\phi, p)
+\end{equation}
+
+and $V = \sum^{\infty}_{k=1} V_k$ where
+
+\begin{equation} \label{eq:7}
+V_k = \frac{T(1+\alpha k)\phi^{k(\alpha - 1)}(p-1)^{\alpha k}}{T(1+k)(p-2)^ky^{\alpha k}}(-1)^k \sin (-k\pi \alpha)
```

Comment:
```suggestion
V_k = \frac{\Gamma(1+\alpha k)\phi^{k(\alpha - 1)}(p-1)^{\alpha k}}{\Gamma(1+k)(p-2)^ky^{\alpha k}}(-1)^k \sin (-k\pi \alpha)
```
Same reason as above, accept this change iff @wendycwong agrees.

Reply:
agree